pydantic/fields.py

```diff
@@ -393,7 +399,32 @@ def Field(
     :param json_schema_extra: extra dict to be merged with the JSON Schema for this field
     :param strict: enable or disable strict parsing mode
     :param validate_default: whether the default value should be validated for this field
+    :param const: removed, use ``Literal`` instead'
+    :param unique_items: removed, use ``Set`` instead
+    :param allow_mutation: deprecated, use ``frozen`` instead
+    :param regex: removed, use `Pattern` instead
+    :param **extra: removed
     """
+    # Check deprecated & removed params of V1.
+    # This has to be removed deprecation period over.
+    if const:
+        raise ValueError('`const` is removed. use `Literal` instead')
+    if min_items:
+        warn('`min_items` is deprecated and will be removed. use `min_length` instead', DeprecationWarning)
+        min_length = min_items
+    if max_items:
+        warn('`max_items` is deprecated and will be removed. use `max_length` instead', DeprecationWarning)
+        max_length = max_items
+    if unique_items:
+        raise ValueError('`unique_items` is removed. use `Set` instead')
```

Comment:
```suggestion
        raise ValueError('`unique_items` is removed, use `Set` instead')
```

Might be worth adding `(this feature is discussed in https://github.com/pydantic/pydantic-core/issues/296)`

https://github.com/pydantic/pydantic-core/issues/296